smoke-tests/src/test/java/com/splunk/opentelemetry/SpringBootSmokeTest.java

```diff
@@ -28,7 +28,9 @@
 class SpringBootSmokeTest extends SmokeTest {
 
   private String getTargetImage(int jdk) {
-    return "open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk" + jdk + ":latest";
+    return "open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk"
+        + jdk
+        + ":20201105.347264626";
```

Comment:
I'm not entirely sure that this change is OK, but it seems that the `latest` image is simply too new, something related to @WithSpan must have changed.

Reply:
Yes, it got moved to a different package, and a different artifact in the past couple of weeks.